config/crd/patches/validation_in_linodevpcs.yaml

```
@@ -0,0 +1,25 @@
+# The following patch adds additional constraints after the built-in name validation for the CRD
+- op: add
+  path: /spec/versions/0/schema/openAPIV3Schema/properties/metadata/properties
+  value:
+    name:
+      type: string
+      x-kubernetes-validations:
+      - rule: 1 <= size(self) && size(self) <= 64
+        message: >-
+          custom validation:
+          linode vpc: labels must be between 3..64 characters
```

```suggestion
          linode vpc: labels must be between 1..64 characters
```
You might have missed this :)

---

👁️👄👁️ Thanks!